### PR TITLE
Correct link to JS SDK

### PR DESCRIPTION
**Pull request by fasiha**
You want to go to http://docs.noms.io, not into the docs.noms.io/ subdirectory in the git repo!

### CODE REVIEW

File: doc/js-tour.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This is a short introduction to using Noms from JavaScript. It should only take a few minutes if you have some familiarity with JavaScript and Node.
 
-During the tour, you can refer to the complete [JavaScript SDK Reference](docs.noms.io/js/) for more information on anything you see.
+During the tour, you can refer to the complete [JavaScript SDK Reference](http://docs.noms.io/js/) for more information on anything you see.
```
Comment (arv) [inline doc/js-tour.md]: Prefer https

**Comment by arv** [comment]
Thanks @fasiha 

Thanks for contributing to Noms. For legal reasons we need you to sign a CLA. See https://github.com/attic-labs/noms/blob/master/CONTRIBUTING.md for details and links.

**Comment by fasiha** [comment]
I’ll ask employer if I should sign individual or corporation CLA but, well, do you just want to make this microscopic documentation change yourself 😇?

**Comment by arv** [comment]
I'm fine making this change myself if you prefer that?

**Comment by fasiha** [comment]
If it’s not too much trouble 🙇 thank you, and thank you for `noms`!

**Comment by arv** [comment]
Completely understand.

Fixed in https://github.com/attic-labs/noms/commit/431a258af5ce2305f85866b693dd8e70423be70d

Thanks again

**Pull request closed without merging**